src/js/chartAuthoring.js

```diff
@@ -26,18 +26,56 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             },
             dataEntryPanel: {
                 type: "floe.chartAuthoring.dataEntryPanel",
-                createOnEvent: "onTemplatesLoaded",
-                container: "{chartAuthoring}.container",
+                createOnEvent: "onPieChartReady",
+                container: "{dataEntryPanel}.container",
```

Comment:
There should be a ```selectors``` block that defines selectors used by parts of this component. An example: https://github.com/fluid-project/chartAuthoring/blob/master/src/js/legend.js#L44-L51